src/style-config/general.scss

```diff
@@ -18,3 +18,11 @@ $app-max-width: 1600px !default;
 $app-width: 958px !default;
 $app-tooltip-pointer-size: 7px !default;
 $app-navigation-height: 40px !default;
+
+
+// Allow extra padding for homepage with responsive navigation
+.demo-header + .sectioniser {
```

Comment:
this isn't the right place for this, it'll end up duplicated everywhere this file is imported and could affect the app